egs/swbd/s5c/local/nnet3/run_reduced_frame.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# This scripts does nnet3-tdnn training and decoding 
```

Comment:
It's also not ideal that this script uses the older config generation and not the xconfigs.  When you have a chance, work with vimal to get an xconfig-based version of this script; you can give it a different name, like 1a->1b.